doc/about/user-guidelines.md

```diff
@@ -9,7 +9,195 @@ hesitate to [open an issue](https://github.com/jupyterhub/mybinder.org-deploy/is
+time someone clicks a Binder link.
+
+1. A Docker image for the link must exist in Binder's image registry. If an image
+   for the current `ref` of the repository *doesn't* exist, one will be built
+   and registered automatically using `repo2docker`. If your
+   configuration files specify a large or complex environment, this will take
+   some time while your image builds.
+2. The Docker image must exist on the node that the user will use. If it does not,
+   then BinderHub will pull the image. If the image is large, this will
+   take some time depending on the server load and image size.
+3. A pod for the user must be created to serve this Docker image. This usually
+   happens in seconds, though may take longer if the server is under a heavy
+   load.
+
+These three things happen in a nested fashion. "3" always happens, "2" only
+happens the *first* time a node is used to serve a particular Docker image, "1"
+only happens the first time someone clicks a Binder link for a repository with
+an updated `ref`. They take roughly decreasing amounts of time to complete,
+so 1 >> 2 >> 3 in terms of how long each operation takes.
+
+If Binder sessions take a while to start, but you know that your image has
+already been built, there's a good chance you are in step 2, and the server is
+still pulling the image onto the node that you'll be using. Please be patient!
+
+### Will repos with fewer notebooks launch faster? Should I split my notebooks into smaller repos?
+
+Number of notebooks in a repo shouldn't have any impact on binder launch time,
+assuming a docker image for that repo is already built. It is worth noting, however,
+that there is a limit to the number of instances of a repository that can be active
+at any moment in time.
+
+### Will repos that are launched often get prioritized and launch faster?
+
+There isn't any intentional prioritization for repos that are launched frequently,
+however, in practice the repos that launch more often will tend to launch faster.
+This is because if a user pod is put on a node that doesn't already have the Docker
+image for that repo, then it'll have to do a Docker pull first, which takes time. If
+a repo is launched a lot, then most likely it will already be on a given node.
+
+
+## Funding and support
+
+### Is `mybinder.org` free to use?
+
+Yes! Though note that it has {ref}`limited computational resources
+<user_memory>`.
+
+### How much does running `mybinder.org` cost?
+
+If you're interested in the technical costs of running
+`mybinder.org`, we publish a semi-up-to-date dataset of our costs at the
+[binder-data](https://github.com/jupyterhub/binder-data/tree/master/billing/data/proc)
+repository. In addition, you can explore these costs with the binder link below!
+
+```{image} https://mybinder.org/badge_logo.svg
+:target: https://mybinder.org/v2/gh/jupyterhub/binder-billing/master?urlpath=lab/tree/analyze_data.ipynb
+```
+
+### How can `mybinder.org` be free to use?
+
```

Comment:
```suggestion
mybinder.org relies on the generosity of donors. If you would like to make a donation see {ref}`support`.

```
Not sure if this will work, or if it should be `[...](../support)`